app/javascript/src/components/Dashboard/Contacts/constants.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Typography, Dropdown, Avatar } from "neetoui";
+
+const MenuIcon = () => (
+  <Dropdown buttonStyle="text" icon={MenuHorizontal}>
+    <li>Edit</li>
+    <li>Delete</li>
+  </Dropdown>
+);
```

Comment:
The component name doesn't resonate with the content. I think calling this component something like `ActionDropdown` would be better.

Reply:
Yes. `ActionDropdown` suits the component.